<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaNovaPlaylist</class>
 <widget class="QDialog" name="JanelaNovaPlaylist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nova Playlist</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconspotper.qrc">
    <normaloff>:/new/prefix1/imgs/spot.png</normaloff>:/new/prefix1/imgs/spot.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	font: 75 16pt &quot;Bahnschrift&quot;;
	background-color:#44c767;
	border-radius: 10px;
	border:1px solid #18ab29;
	color:#ffffff;
	font-size:17px;
}
QPushButton:hover {
	background-color: #53f37e;
}
QDialog{
	background-color: #303030;
}
QLabel{
	font: 75 16pt &quot;Bahnschrift&quot;;
	color:#ffffff;
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Nome Playlist:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditNomePlaylist"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnCriarPlaylist">
       <property name="text">
        <string>CRIAR PLAYLIST</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Faixas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidgetFaixas"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddAPlaylist">
       <property name="text">
        <string>ADICIONAR A PLAYLIST</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Albuns&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidgetAlbuns"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnPesquisarFaixas">
       <property name="text">
        <string>PESQUISAR FAIXAS</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="iconspotper.qrc"/>
 </resources>
 <connections/>
</ui>
